docs/mockups/background.html

```diff
@@ -76,8 +76,8 @@
       </span>
      </button>
      <a class="text-nowrap" href="home.html" style="text-decoration: none">
-      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" src="../www/img/logo_transparent.png" style="height:50px;" title="VAPALS-ELCAP"/>
-      <img alt="VAPALS-ELCAP Logo" src="../www/img/logo_name.png" style="height:50px;" title="VAPALS-ELCAP"/>
+      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" src="../www/img/logo_transparent.png" title="VAPALS-ELCAP" width="50"/>
```

Comment:
I assume that you meant to change height to width here, when you changed attributes.

Reply:
disregard. rebaseing to master is causing some issues. It will merge correctly once I fix that.

Reply:
Okay.